projects/schizophrenia/src/HomePage.js

```diff
@@ -91,37 +93,50 @@ const HomePage = ({ setCurrentGene }) => {
       <TitleGroup>
         <Title>SCHEMA</Title>
         <Subtitle>Schizophrenia exome meta-analysis consortium</Subtitle>
-        <ParagraphText>Lorem ipsum dolor sit amet, consectetur adipisicing elit.
-          Et corrupti dolore qui debitis voluptatem consectetur doloribus repellendus
-          soluta atque consequuntur molestias dolor porro, rem, pariatur minus
-          eligendi vero dolorum recusandae incidunt sint laborum. Delectus ipsa
-          amet, nobis.</ParagraphText>
+        <ParagraphText>The Schizophrenia Exome Sequencing Meta-analysis (SCHEMA) consortium is a large multi-site collaboration dedicated to aggregating, generating, and analyzing high-throughput sequencing data of schizophrenia patients to improve our understanding of disease architecture and advance gene discovery. The consortium was formed in mid-2017 with a deep commitment of data sharing, diversity, and inclusivity - our hope is that the findings from this study and others like it will provide a foundation for further investigation of disease mechanism and therapeutic discovery. This browser is part of that overall effort to display and share these results with the wider scientific community.</ParagraphText>
+
+<ParagraphText>To date, the SCHEMA consortium have have sequenced and processed the whole exomes of over 25,000 schizophrenia cases and 50,000 matched controls using a standardized protocol, yielding one of the largest sequencing data sets of a complex trait to date. Our study has actively recruited from diverse global populations, and includes individuals of European, Latin American, East Asian, Ashkenazi Jewish, and African American ancestry. Because the sequence data was generated with various capture technologies over a span of seven years, we adapted and developed methods to reduce possible confounders, and incorporated this information during the quality control and analysis steps. The first results have provided genome-wide significant results associating rare variants in individual genes to risk of schizophrenia, and later releases are planned with larger number of samples that will further increase power.</ParagraphText>
+
+<ParagraphText>The SCHEMA consortium is made possible by the generosity of many funders, including the Stanley Foundation, and NIH, and the leadership of its members. The principal investigators and groups who have contributed to this current release are listed here. We would also like to thank the many tens of thousands of patients and families who generously contributed to our effort.</ParagraphText>
+
+
+
         <SearchContainer>
         </SearchContainer>
         <Announcements>
           <AnnouncementsTitle>Announcements</AnnouncementsTitle>
           <Announcement>
             <AnnouncementHeader>
-              <em>October 17, 2017</em>. Tempora,
-              possimus, iusto. Rem explicabo, consequuntur cumque id itaque deserunt,
-               harum provident?
+              <em>March, 2018</em>.
```

Comment:
`AnnouncementHeader` has a `font-weight: bold` style... do we need the `<em>` elements as well?